pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = { text = "GPL" }
 classifiers = [ "Programming Language :: Python :: 3" ]
 urls = { documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_IntegrationTests" }
 dynamic = [ "version" ]
-dependencies = [ "yamllint" ]
+dependencies = [ "yamllint", "parameterized" ]
```

Comment:
Aren't these really dev dependencies and not runtime ones? If they are, we should make them so.

Reply:
I think I got it.  At least mostly.